submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/pagetab/FirePageTabService.kt

```diff
@@ -21,53 +20,54 @@ class FirePageTabService(
         val subFiles = serializationService.generateSubPageTab(sub, fireTempFolder)
         val fileListFiles = serializationService.generateFileListPageTab(sub, fireTempFolder)
 
-        val section = process(sub.section) { updateFileList(it, sub.filesPath, fileListFiles) }
+        val section = process(sub.section) { updateFileList(it, sub.relPath, fileListFiles) }
 
         return when {
-            section.changed -> sub.copy(pageTabFiles = extFiles(sub.filesPath, subFiles), section = section.section)
-            else -> sub.copy(pageTabFiles = extFiles(sub.filesPath, subFiles))
+            section.changed -> sub.copy(pageTabFiles = subExtFiles(subFiles, sub.relPath), section = section.section)
+            else -> sub.copy(pageTabFiles = subExtFiles(subFiles, sub.relPath))
         }
     }
 
     private fun updateFileList(sec: ExtSection, path: String, tab: Map<String, TabFiles>): Section {
```

Comment:
Sorry I didn't catch this in previews reviews but I think the `TabFiles` class should be called `PageTabFiles`